app/src/main/java/com/example/makeitso/model/service/impl/StorageServiceImpl.kt

```diff
@@ -38,6 +39,7 @@ class StorageServiceImpl
 constructor(private val firestore: FirebaseFirestore, private val auth: AccountService) :
   StorageService {
 
+  @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
Which API is experimental here? And why did we not need this before?

Reply:
`flatMapLatest` throws a warning if we don't add this `@OptIn` - it should've been added when we added `flatMapLatest`, but we let it slip.